roles/tpa_single_node/vars/main.yml

```diff
@@ -18,46 +18,48 @@ tpa_single_node_pg_user: guac
 tpa_single_node_pg_ssl_mode: disable
 
-# Storage Service (S3 or Minio)
+# Storage Service - S3 or Minio
 tpa_single_node_storage_endpoint: # <Enter minio storage endpoint, For AWS this field is not necessary>
-tpa_single_node_storage_region: # <AWS S3 Storage region> # For Minio just keep us-west-1
+tpa_single_node_storage_region: eu-west-1 # <AWS S3 Storage region> - For Minio just keep us-west-1
 tpa_single_node_storage_access_key: "{{ lookup('env', 'TPA_STORAGE_ACCESS_KEY') }}"
 tpa_single_node_storage_secret_key: "{{ lookup('env', 'TPA_STORAGE_SECRET_KEY') }}"
-tpa_single_node_storage_bombastic_bucket: bombastic-default # <bombastic storage bucket name>
-tpa_single_node_storage_v11y_bucket: v11y-default # <v11y storage bucket name>
+tpa_single_node_storage_bombastic_bucket: bombastic-rhtpa
+tpa_single_node_storage_vexination_bucket: vexination-rhtpa
+tpa_single_node_storage_v11y_bucket: v11y-rhtpa
 
-
-# SQS_SERVICE
-tpa_single_node_event_bus_type: # Either kafka or sqs
+# QUEUE Service - Kafka or SQL
+tpa_single_node_event_bus_type: sqs # Either <kafka> or <sqs>
 tpa_single_node_event_access_key_id: "{{ lookup('env', 'TPA_EVENT_ACCESS_KEY_ID') }}" # Kafka Username or AWS SQS Access Key ID
 tpa_single_node_event_secret_access_key: "{{ lookup('env', 'TPA_EVENT_SECRET_ACCESS_KEY') }}" # Kafka password or AWS SQS Secret Access Key
+tpa_single_node_event_endpoint: "{{ lookup('env', 'TPA_EVENT_ENDPOINT') }}"
 
-### Kafka fields
-# <Only needed for Kafka service, For AWS this field is not necessary>
+### Kafka - Only needed for Kafka service
 tpa_single_node_kafka_bootstrap_servers: "{{ lookup('env', 'TPA_EVENT_BOOTSTRAP_SERVER') | default('tpa_single_node_default_empty', true) }}"
 # Security protocol used for Kafka user authentication.
-tpa_single_node_kafka_security_protocol: # Example - SASL_PLAINTEXT
+tpa_single_node_kafka_security_protocol: SASL_PLAINTEXT
 # Kafka Authentication Mechanism which is spec.kafka.listeners.authentication.type in Block letters.
-tpa_single_node_kafka_auth_mechanism: # Example - SCRAM-SHA-512
+tpa_single_node_kafka_auth_mechanism: SCRAM-SHA-512
 
 ### AWS SQS fields
-tpa_single_node_sqs_region: # AWS SQS Region
+tpa_single_node_sqs_region: eu-west-1 # AWS SQS Region
 
 ### Event topics
-tpa_single_node_bombastic_topic_failed: # Bombastic Events topic failed
-tpa_single_node_bombastic_topic_indexed: # Bombastic Events topic indexed
-tpa_single_node_bombastic_topic_stored: # Bombastic Events topic stored
-tpa_single_node_vexination_topic_failed: # Vexination Events topic failed
-tpa_single_node_vexination_topic_indexed: # Vexination Events topic indexed
-tpa_single_node_vexination_topic_stored: # Vexination Events topic stored
-tpa_single_node_v11y_topic_failed:  # v11y Events topic failed
-tpa_single_node_v11y_topic_indexed:  # v11y Events topic indexed
-tpa_single_node_v11y_topic_stored:  # v11y Events topic stored
+tpa_single_node_bombastic_topic_failed: bombastic-failed-rhtpa
```

Comment:
@desmax74, 
Those are configuration dependent and I'm going to remove them.
Meanwhile this is annoying because a value is required.
So we could leave a default value such as :
`bombastic-failed-default`
or leave them empty